### DISCUSSION TITLE
Ability to Edit chrome://flags for experimental features

### DISCUSSION THREAD

**Discussion by mpgithubcode** [minbrowser/min#2019] [General]
Would love to be able to enable things like hardware acceleration and various other features to speed up performance

**Comment by unrealbanter** [comment]
Correct me if I'm wrong, even though this is kind of an old post, I believe it's on by default being that my GPU is handling quite a bit of tasks.

**Comment by morellicai** [comment]
I know it's not quite the discussion here, but I see that it fits. I use hyprland, and I really needed to be able to change settings in the chrome://flags to be able to change the browser's flag to recognize that it is in wayland so as not to have visual bugs like this in the image I attached here.
<img width="3840" height="1080" alt="screenshot" src="https://github.com/user-attachments/assets/eb9a30fc-61e4-4c0a-963d-913863cd8a4f" />
In the screenshot I took of my two screens, you can see that a window opens on my other screen when I press the 3 dots that are in the left corner of the browser. Without being able to configure flag options, I can't remove this "bug" that is the incompatibility with applications in wayland

**Comment by ariadesu** [comment]
+1, for me it's to be able to disable the over-scroll history gesture on touchpad.

**Comment by okyr-ux** [comment]
I'd love to enable the Split View flag. i tried:
* !flags
* about:flags
* min://flags
* about:experiments
* min://experiments

**Comment by morellicai** [comment]
I found a good way to update the flags without having to go into the settings. We can define the flags in a file in the ".config". I named it as "chromium-flags.conf", here it worked fine.
